src/evm/pragma.rs

```diff
@@ -0,0 +1,25 @@
+use alloy::network::Ethereum;
+use alloy::providers::fillers::{ChainIdFiller, FillProvider, GasFiller, JoinFill, NonceFiller};
+use alloy::providers::{Identity, RootProvider};
+use alloy::transports::http::{Client, Http};
+use alloy::{providers::fillers::BlobGasFiller, sol};
+
+sol!(
+    #[allow(missing_docs)]
+    #[sol(rpc)]
+    Pragma,
```

Comment:
You can remove the ABI from the code and instead just define the interface, example here:
https://github.com/astraly-labs/pragma-monorepo/blob/dev/more_networks/rust/theoros/src/rpc/evm/hyperlane.rs